static/js/app.js

```diff
@@ -49,6 +49,31 @@ form?.addEventListener('submit', (event) => {
 }
 );
 
+
+
+// share on twitter
+function shareOnTwitter() {
+  var twitterWindow = window.open('https://twitter.com/share?url=' + document.URL, 'twitter-popup', 'height=350,width=600');  
+  if (twitterWindow.focus) { twitterWindow.focus(); }
```

Comment:
```suggestion
  if (twitterWindow.focus) {
      twitterWindow.focus(); 
      }
```

Reply:
I thought the formatting may look a little more consistent if the action is on a separate line.